src/RahulH/Q1.java

```diff
@@ -0,0 +1,36 @@
+package RahulH;
+
+public class Q1 {
+	
+	int num[] = {1, 2, 3, 4, 5, 6, 7, 9, 10};
+	int sum1 = 0;
+	int sum2 = 55;
+	int missing;
```

Comment:
Hi @RahulHAug19 
This is not correct way to idnetify missing number...
go with this belwo logic
n*(n+1)/2